src/events/_event-post-macros.html

```diff
@@ -354,13 +371,13 @@ <h2 class="event-venue_heading">{{ city }}</h2>
                   {% endif %}
                 </td>
                 <td class='{{ options.speaker_col_classes }}' data-label="Speakers">
-                    {% for speaker in block.speakers %}
-                        {%- if speaker.url -%}
-                            <a href="{{ speaker.url }}">{{ speaker.name }}, {{ speaker.position }}</a>{{ ';' if loop.index < loop.length }}
-                        {% else %}
-                            {{ speaker.name }}, {{ speaker.position }}{{ ';' if loop.index < loop.length }}
-                        {%- endif -%}
-                    {% endfor %}
+                {% for speaker in block.speakers %}
+                    {%- if speaker.url -%}
+                        <a href="{{ speaker.url }}">{{ speaker.name }}, {{ speaker.position }}</a>{{ ';' if loop.index < loop.length }}
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/38a787781f8ef2dcd1d5aaebedd239e3a388d9cd